cohesity_management_sdk/http/auth/auth_manager.py

```diff
@@ -43,6 +43,10 @@ def authorize(cls, Configuration):
         if Configuration.domain is not None:
             body.domain = Configuration.domain
 
+        if Configuration.otp_type:
+            body.otp_type = Configuration.otp_type
+            body.otp_code = Configuration.otp_code
+
         auth_controller = AccessTokensController(Configuration)
         token = auth_controller.create_generate_access_token(body)
```

Comment:
Great commit @korzorro, body class needs those params as well. Can you fix those ? 
Since these are optional, we should be ok to merge it into master, we will review in detail and merge the fix once its done.

Reply:
@pyashish In this snippet, `body` is an instance of `AccessTokenCredential` which already has the member variables `otp_type` and `otp_code`. If you're referring to a different class, let me know which one you would like edited.

Reply:
That's right , my bad. Although this should be part of the Client initialization request as optional params. Currently,  one needs to initialize client and then set client.config.otp_code and client.config.otp_type correct ? 
Did you try out a workflow ?

Reply:
You are correct. You would have to set those two vars post initialization. I have this fork in production and the MFA is working with TOTP. I will add the optional params to CohesityClient, but as is, the manual setting of `otp_...` gets the job done.

Again, mind that this requires that the user manages the OTP.

Reply:
Sorry for the delay, HNY. 

Yes, that's accurate, we are taking a looking to make it part of client config , this would be building blocks for this.